data/hai/hai missions.txt

```diff
@@ -209,6 +209,69 @@ mission "Ask the Hai about the Unfettered"
+			`	He looks at you very seriously to make sure what he's said has sunk in.`
+			choice
+				`	"I understand, I've stepped on some toes."`
+					goto trouble
+					to display
+						"reputation: Hai" < 1
+				`	"I understand, thank you for warning me."`
+					goto end
+					to display
+						"reputation: Hai" >= 1
+				`	"Why should I get in trouble for defending myself against Sympathizers."`
+			`	He shrugs. "You should try not to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`
+			`	"When a report gets made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. As you are not a citizen though, the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a Captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`
+			`	"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."`
+			
+			label trouble
+			branch "less trouble"
+				"reputation: Hai" >= 1
+				"reputation: Hai (Wormhole Access)" >= 1
+				"reputation: Hai Merchant" >= 1
+				"reputation: Hai Merchant (Human)" >= 1
+			# At least one other group besides the Sympathizers are upset.
+			`	"You have gotten yourself in trouble Captain."`
+			action
+				"reputation: Hai" >?= 1
+				"reputation: Hai (Wormhole Access)" = "reputation: Hai"
+				"reputation: Hai Merchant" >?= 1
+				"reputation: Hai Merchant (Human)" >?= 1
+			`	He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."`
+			
+			label "less trouble"
+			# Only the Sympathizers are upset.
+			`	"So far, you have been lucky. Hai Security has taken your history into account, and decided your recent activities are within the tolerable range of responses to the hostile situations you have got yourself into..." He slides a data stick across to you. "Nevertheless, this is a copy of the file on you that states you have now been advised of the situation. If you now proceed to upset Hai Security, they will not provide you with a warning."`
+			
```

Comment:
I think you forgot to repair sympathizer reputation here?
or do you never repair that rep intentionaly?

Reply:
I'm on the fence about it, leaning towards intentionally not repairing it, since you'll only be negative by a very small amount for this option to appear.

Reply:
you could just give +50 relations with them or smth

Reply:
If it's small enough to be fixed by +50 relations, that's the scenario when I *don't* think anything should be done...

Reply:
What about cutting the rep in two? That way not much changes if its low but otherwise it does more
We could always check for it being lower than smth too

Reply:
If it's really low and Hai rep isn't - that's a different mission I want to make.